### DISCUSSION TITLE
macOS Firefox

### DISCUSSION THREAD

**Discussion by ChasCB** [black7375/Firefox-UI-Fix#756] [Q&A] [answered]
Does your theme work on macOS Firefox ??
If so, will it conflict with [CustomCSSforFx](https://github.com/Aris-t2/CustomCSSforFx) and by own userChrome.css ??

A long, long time ago I was an avid user of Mostly Crystal by Cat Thief, but her work has now gone to cat heaven, so sad because it was fabulous.

**Comment by black7375** [comment, marked as answer]
This project should be works at MacOS.
However, there may be a conflict with CustomCSSforFx.
Unfortunately, conflicts can occur as both projects have a large CSS size and a large custom area.

**Reply by ChasCB** [reply]
Ok thanks, also thanks for your very quick reply. I will give it a go soon and get back to you.
